packages/web-components/src/radio/radio.styles.ts

```diff
@@ -2,141 +2,146 @@ import { css } from '@microsoft/fast-element';
 import { display, forcedColorsStylesheetBehavior } from '../utils/index.js';
 import {
   borderRadiusCircular,
-  borderRadiusSmall,
-  colorCompoundBrandForeground1,
-  colorCompoundBrandForeground1Pressed,
-  colorCompoundBrandStrokeHover,
-  colorCompoundBrandStrokePressed,
-  colorNeutralForeground2,
-  colorNeutralForeground3,
-  colorNeutralForegroundDisabled,
+  borderRadiusMedium,
+  colorCompoundBrandBackground,
+  colorCompoundBrandBackgroundHover,
+  colorCompoundBrandBackgroundPressed,
+  colorCompoundBrandStroke,
+  colorNeutralBackground1,
+  colorNeutralBackgroundDisabled,
+  colorNeutralForegroundInverted,
   colorNeutralStrokeAccessible,
   colorNeutralStrokeAccessibleHover,
   colorNeutralStrokeAccessiblePressed,
-  colorStrokeFocus1,
+  colorNeutralStrokeDisabled,
   colorStrokeFocus2,
-  fontFamilyBase,
-  fontSizeBase300,
-  fontWeightRegular,
-  lineHeightBase300,
-  spacingHorizontalS,
-  spacingHorizontalXS,
-  spacingVerticalS,
+  colorTransparentStroke,
+  strokeWidthThick,
+  strokeWidthThin,
 } from '../theme/design-tokens.js';
 
-/** Radio styles
+const checkedState = css.partial`:is([state--checked], :state(checked))`;
+
+const disabledState = css.partial`:is([state--disabled], :state(disabled))`;
```

Comment:
`disabledState` and `checkedState` appear to be exported from the styles utilities now.

Reply:
`checkedState` and `disabledState` are now imported from the states module.